conda/plugins/manager.py

```diff
@@ -169,9 +184,7 @@ def get_plugin_manager() -> CondaPluginManager:
     plugin_manager = CondaPluginManager()
     plugin_manager.add_hookspecs(CondaSpecs)
     plugin_manager.load_plugins(
-        solvers,
-        *virtual_packages.plugins,
-        *subcommands.plugins,
+        solvers, *virtual_packages.plugins, *subcommands.plugins
```

Comment:
```suggestion
        solvers,
        *virtual_packages.plugins,
        *subcommands.plugins,
```

Reply:
@kenodegard, I prefer this but `black` does not 😭. Let's just keep what the code formatter wants.

Reply:
its all about the magic trailing comma: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma